caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java

```diff
@@ -153,7 +153,7 @@ public void testInvalidTenantId() throws Exception
         }
     }
 
-    @Test(expected = ElasticsearchStatusException.class)
+    @Test(expected = OpenSearchStatusException.class)
     public void testStringLengthRestrictionTenantId() throws Exception
```

Comment:
Could not spot where this is thrown from

Reply:
This isnt a test I've written, however previously it was an ElasticsearchStatusException, which dosnt exist in Opensearch, so I basically switched it to the OS equivalent